### PR TITLE
Otel payloads examples implementation

### PR DESCRIPTION
**Pull request by pmatyjasek-sumo**
Otel payloads examples implementation

Signed-off-by: Patryk Matyjasek <pmatyjasek@sumologic.com>

### CODE REVIEW

File: examples/demo-app/Makefile [hunk 1]
```diff
@@ -0,0 +1,2 @@
+build:
+	go build .
```
Comment (pmalek-sumo) [inline examples/demo-app/Makefile:2]:
```suggestion
	go build .

```

File: examples/demo-app/main.go [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+	controller "go.opentelemetry.io/otel/sdk/metric/controller/basic"
+	processor "go.opentelemetry.io/otel/sdk/metric/processor/basic"
+	"go.opentelemetry.io/otel/sdk/metric/selector/simple"
+	"go.opentelemetry.io/otel/sdk/resource"
+	sdktrace "go.opentelemetry.io/otel/sdk/trace"
+	"go.opentelemetry.io/otel/semconv"
+	"google.golang.org/grpc"
+)
+
+func initProvider() func() {
```
Comment (pmalek-sumo) [inline examples/demo-app/main.go:42]: Comment: what function does it return? What it does? When to call it?

File: examples/demo-app/main.go [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+	otel.SetTextMapPropagator(propagation.TraceContext{})
+	otel.SetTracerProvider(tracerProvider)
+	global.SetMeterProvider(cont.MeterProvider())
+	handleErr(cont.Start(context.Background()), "failed to start metric controller")
+
+	return func() {
+		handleErr(tracerProvider.Shutdown(ctx), "failed to shutdown provider")
+		handleErr(cont.Stop(context.Background()), "failed to stop metrics controller") // pushes any last exports to the receiver
+		handleErr(exp.Shutdown(ctx), "failed to stop exporter")
+	}
```
Comment (pmalek-sumo) [inline examples/demo-app/main.go:89]: This is meant to be a shutdown/cleanup func? Why mix `context.Background()` with `ctx`

**Review by pmalek-sumo — COMMENTED** [review]
Can we add a README describing what this tool does? It's already landing in a separate dir so it would be a perfect place for that I guess

**Comment by astencel-sumo** [comment]
Closing this as the backend team declared they don't need this now.

**Pull request closed without merging**